### ISSUE TITLE
Audio components improvement

### ISSUE THREAD

**Issue by fcoveram** [WordPress/openverse#2910]
## Problem

From the audio control update requested in #2602, the following components need to be updated to incorporate the button in a clean way.

- Box (large, and medium)
- Row (large, medium, and small)
- Global

## Proposal

All components and their variants mentioned above have new spacing, size, and style values. Before diving into each, here is a set of all pages involved in the component update.

### Search results. All content

![All content search results in all breakpoints](https://github.com/WordPress/openverse/assets/895819/3861b8dc-268c-422f-8637-d6091c300153)

### Search results. Audio

![Audio search results in all breakpoints](https://github.com/WordPress/openverse/assets/895819/d6ed4ce3-fcd8-4bf4-85ad-3f6b40abb00a)

### Single result page

![Single result page mockups in all breakpoints](https://github.com/WordPress/openverse/assets/895819/c818f5ae-6984-4786-ae83-5d8abec9f6df)

### Audio box

The new box audio component has the following main changes:
- The component was reduced to two variants: Large and Small. The medium size was not needed for Figma mockups.
- The audio control is `size: small` .
- Both large and small variants allow users to control the audio with the current play, pause, and repeat actions.
- Only large variant shows the wave area. 
- All texts styles were updated to be consistent with other library changes.

| Small | Large |
|---|---|
| <img src="https://github.com/WordPress/openverse/assets/895819/9a8bb8d6-b75e-4281-928a-359e8a3646ae" alt="Small variant of audio box" width="230"> | <img src="https://github.com/WordPress/openverse/assets/895819/1b8b1ab0-d494-4aa5-9186-9aee35160da4" alt="Large variant of audio box" width="230"> |

### Audio row

For the main audio component, the following changes were applied:

**Size: Large**

- The current variant named “Medium” is now “Large”. This change comes from the variant used per breakpoint.
- The audio control is `size: large`.
- Cover image is now `64px` to match the audio control button
- All texts styles were updated to be consistent with other library changes.
- Spacings in the audio text info were updated.

![Large variant of audio row component](https://github.com/WordPress/openverse/assets/895819/ecb70726-01f9-4489-8e17-f78c2a60f956)

**Size: Medium**

- The current variant named “Large” is now “Medium”. This change comes from the variant used per breakpoint.
- The audio control is `size: medium`.
- Cover image is the height formed by the sum of text info and wave area. In Figma it was set as `95px`, but this should be revisited in code to meet a visual balance.
- All texts styles were updated to be consistent with other library changes.
- Spacings in the audio text info were updated.

<img width="600" alt="Medium variant of audio row component" src="https://github.com/WordPress/openverse/assets/895819/e8f409fe-9f98-429f-8e89-2b88db4e90f9">

**Size: Small**

- The current variant named “Large” is now “Medium”. This change comes from the variant used per breakpoint.
- The audio control is `size: medium`.
- Audio control is now placed in the left-bottom area to be consistent with the Audio box action.
- Cover image is the height formed by the sum of text info and wave area. In Figma it was set as `95px`, but this should be revisited in code to meet a visual balance.
- All texts styles were updated to be consistent with other library changes.
- Spacings in the audio text info were updated.

<img width="230" alt="Small variant of audio row component" src="https://github.com/WordPress/openverse/assets/895819/c9eaaef3-8154-41fa-9eff-cfe5f04769c8">

### Global player

- Cover image is now `48px` to meet other library changes.
- The audio control is `size: medium`.
- There is a new style for the component border.
- In page context, the component has a new position to prevent the blending effect with the browser area. See the mockups shared above.
- There is a new style for long audio titles.
- The current style for playing changed to solve interaction issues.

<img width="420" alt="Global player component" src="https://github.com/WordPress/openverse/assets/895819/11343234-17ed-4fe4-b93e-e1822db3c0a3">

## Mockups

Figma file: [Audio components improvement](https://www.figma.com/file/LMGuwErTWIYkOIEZI4t320/Audio-components-improvement?type=design&node-id=22-13656&mode=dev)

**Comment by fcoveram** [comment]
In addition to the new styles described above for the global player, here I show three interaction changes that keep the current core idea.

## Closing the component

Closing the component is an [VIconButton](https://docs.openverse.org/storybook/?path=/story/components-viconbutton--default-story&args=variant:transparent-gray) in `size: small / style: transparent / color: gray`.

<img src="https://github.com/WordPress/openverse/assets/895819/0e23631b-bfcb-499a-994b-b8f67c5b9b69" alt="Audio global player. Closing interaction" width="450">

## Seeking audio time

The time-seeking interaction is now only in the wave area. The current player uses the whole yellow area to interact with time but it impedes going to the audio details page.

<img src="https://github.com/WordPress/openverse/assets/895819/41f595b3-5c9a-409f-93ac-7cb690307703" alt="Audio global player. Seeking time" width="450">

## Going to audio single page

In line with the previous point, audio title is now a link to its single result page. In `hover` the link is underlined, like a [regular link](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?type=design&node-id=3667-9753&mode=dev).

<img src="https://github.com/WordPress/openverse/assets/895819/37daac80-2482-471f-90f8-48231393627b" alt="Audio global player. Hover interaction of audio title" width="450">

## Component background color

Finally, reinforce that the component background is white. This fixes a style missed in the [initial design](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?type=design&node-id=133-13723&mode=design&t=CO20svp8LyZLuQtG-4).
